vm/statement_test.go

```diff
@@ -116,6 +116,56 @@ func TestDefStatement(t *testing.T) {
 	}
 }
 
+func TestModuleStatement(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected int
+	}{
+		{`
+		module Bar
+		  def bar
+		    10
+		  end
+		end
+
+		class Foo
+		  include(Bar)
+		end
+
+		Foo.new.bar
+		`,
+			10},
+		{`module Foo
```

Comment:
@adlerhsieh BTW, another thing you can contribute is to design some more complex tests for `rooby` 😀